consensus/istanbul/core/prepare_test.go

```diff
@@ -214,8 +214,8 @@ OUTER:
 			if r0.state != StatePreprepared {
```

Comment:
There's also a condition further above at line 164 that you should check if it needs to be changed (sorry, I don't know how to comment there specifically).